content/en/blog/2023/sig-contribex-spotlight.md

```diff
@@ -0,0 +1,294 @@
+---
+layout: blog
+title: "Spotlight on SIG ContribEx"
+date: 2023-08-22
```

Comment:
```suggestion
date: 2023-08-14
```
If we can get this in before the next release, it'll help. Otherwise, let's wait until the post-release blog series has gone out.

Reply:
Agreed.